pkgs/data/fonts/fixedsys-excelsior/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl } :
+
+let 
+  major = "3";
+  minor = "00";
+  version = "${major}.${minor}";
+
+in
+
+stdenv.mkDerivation rec {
+  name = "fixedsys-excelsior-${version}";
+
+  src = fetchurl {
+    url = http://www.fixedsysexcelsior.com/fonts/FSEX300.ttf;
+    sha256 = "6ee0f3573bc5e33e93b616ef6282f49bc0e227a31aa753ac76ed2e3f3d02056d";
+  };
+
+  phases = [ "installPhase" ];
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/truetype/
+    cp $src $out/share/fonts/truetype/${name}.ttf
+  '';
+
+  meta = {
+    description = "Fixedsys Excelsior is a pan-unicode version of Fixedsys";
```

Comment:
Please do not use the package name in the short description; "foo is a bar" is better as simply "A bar"